foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,84 @@
+console.log(playRound(playerSelection, computerSelection));
+```
 
-   - Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
+#### Step 4: game function
 
-1. **Important note:** you want to `return` the results of this function call, *not* `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
+Your game will play 5 rounds. Write a function named `game` that uses `playRound` to play 5 rounds, keeps score, and declares a winner at the end:
 
-   ```javascript
-   function playRound(playerSelection, computerSelection) {
-     // your code here!
-   }
+1. Create a new function named `game`.
+1. Get the human player choice using the `prompt` method. Read this [MDN article about prompt method](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) learn more about it.
+1. Put the previous `playRound` function inside `game`.
+1. Play 5 rounds by calling `playRound` 5 times.
+   - **Hint**: When you assign a function call to a variable, the return value of that function is assigned to the variable. Accessing the variable afterward will only provide the assigned value; it doesn't recall the function. You need to recall the choice functions to get new choices for each round.
 
-   const playerSelection = "rock";
-   const computerSelection = getComputerChoice();
-   console.log(playRound(playerSelection, computerSelection));
-   ```
+Re-work your previous functions or create more helper functions if necessary. Specifically, you may want to change the return values to something more useful.
 
-1. Write a NEW function called `playGame()`. Use the previous function *inside* of this one to play a five round game that keeps score and reports a winner or loser at the end.
-   - You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-   - At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-   - Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
-   - Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-   - Feel free to create more "helper" functions if you think it would be useful.
+If you already know about loops, you can use them. If not, don't worry! Loops will be covered in the next lesson.
```

Comment:
Not sure if it's worth mentioning this? Not a strong feeling, just wondering if there's really a point to mention it.